packages/calcite-components/src/components/combobox-item/combobox-item.scss

```
@@ -1,3 +1,18 @@
+/**
+ * CSS Custom Properties
+ *
+ * These properties can be overridden using the component's tag as selector.
+ *
+ * @prop --calcite-combobox-text-color: Specifies the text color of the component.
```

Doc consistency suggestion:

```suggestion
 * @prop --calcite-combobox-text-color: Specifies the component's text color.
```